public/v4/logos/baikal.png

```diff
@@ -0,0 +1,1477 @@
+
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
This is not a PNG file, it's an html file!

I assume https://sabre.io/img/baikal.png is the right PNG file?